src/edu/ucsb/nceas/metacat/admin/MetacatAdmin.java

```diff
@@ -89,7 +89,7 @@ public static void updateUpgradeStatus(String propertyName, String status, boole
             //this sub upgrade process failed or is in progress, so the whole process will have the same status as well
             PropertyService.setPropertyNoPersist("configutil.upgrade.status", status);
         } 
-	    if(persist) {
+	    if(persist && Boolean.parseBoolean(System.getenv("METACAT_IS_RUNNING_IN_A_CONTAINER"))) {
```

Comment:
Is there a reason that Metacat doesn't need to persist properties when it is not running in a container?

Reply:
Good Q. Also, the env variable `METACAT_IS_RUNNING_IN_A_CONTAINER` seems far too long and verbose, and will interfere with display of the environment.  Can it be shortened to `IS_CONTAINER` or something more clever?

Reply:
Does that answer your question, @taojing2002 ?

Reply:
And @mbjones - when i rename it, how would you feel about the new name using “k8s” instead of the more generic “container”?

Reply:
Sorry, @taojing2002 - the double negative confused me 😂 - I understand now. The condition should be NOT running in a container. I will change

Reply:
@mbjones - I will rename the `METACAT_IS_RUNNING_IN_A_CONTAINER` env var, but will do it as a separate PR, to keep things simpler, since I'm already working on another branch that uses it too

Reply:
https://github.com/NCEAS/metacat/pull/1693